package.json

```diff
@@ -98,7 +92,6 @@
     "eslint-plugin-prettier": "^4.0.0",
     "husky": "^8.0.3",
     "jest": "^27.4.7",
-    "lerna": "^7.0.2",
```

Comment:
why delete `lerna`?

Reply:
After deleting the docsify-server-render package, we don't need to manage a sub-package anymore.

The funny thing is that, I realized after we merged https://github.com/docsifyjs/docsify/pull/2094 that once we removed `lerna bootstrap`, we are not using `lerna` for anything, so we could simply deleted it. 😄

Reply:
I.e., now the only thing we need to do during development is `npm install` and `npm run whatever`.